server/py/services/api/tests/unit/runtimes/test_serving.py

```diff
@@ -165,23 +160,6 @@ def _assert_deploy_spec_has_secrets_config(
                     },
                 },
             ]
-            if use_config_map:
```

Comment:
Why was this removed? Has this changed due to this PR?

Reply:
This is a unit test, so everything here is a bit complex, mocking flows in various ways. If the functionality is now in the launcher then I can't really point at an easy fix to make sure we go through this flow.